public/app/features/dashboard/components/PanelEditor/DynamicConfigValueEditor.tsx

```diff
@@ -35,7 +35,10 @@ export const DynamicConfigValueEditor: React.FC<DynamicConfigValueEditorProps> =
   // eslint-disable-next-line react/display-name
   const renderLabel = (includeDescription = true, includeCounter = false) => (isExpanded = false) => (
     <HorizontalGroup justify="space-between">
-      <Label category={item.category?.splice(1)} description={includeDescription ? item.description : undefined}>
+      <Label
+        category={item.category?.filter(c => c !== undefined) as string[]}
```

Comment:
Wonder if we should remove this category on label and instead just support nested OptionGroup sections

Reply:
@torkelo I think we should have a fallback for such situation, when someone adds more than two cattegories. Otherwise it may be confusing some of the strings in the list are missing. We could pottentially use a tuple to define the categories array `[string | undefined, string]`

But TBH - not sure if it's worth it.

Reply:
Yea not sure either, let's keep it and revisit as we do https://github.com/grafana/grafana/pull/30329